tests/build/build.go

```diff
@@ -124,9 +130,13 @@ var _ = framework.BuildSuiteDescribe("Build service E2E tests", Label("build", "
 			//Delete the quay image repo since we are setting delete-repo=false
 			_, err = build.DeleteImageRepo(imageRepoName)
 			Expect(err).NotTo(HaveOccurred(), "Failed to delete image repo with error: %+v", err)
+
+			// Delete the sample private image repo created for checking inside build.IsQuayOrgSupportsPrivateRepo()
```

Comment:
Shouldn't the function `IsQuayOrgSupportsPrivateRepo()` clean after itself right after checking?

Reply:
resolving, as Mykola raised the issue previously.

Reply:
Agree will update the function to include cleanup as well.